silkworm/silkrpc/http/request_parser.cpp

```diff
@@ -29,6 +29,19 @@
 
 namespace silkworm::rpc::http {
 
+void RequestParser::reset() {
+    if (last_len_) {
+        last_len_ = 0;
+        delete[] buffer_;
```

Comment:
It is better to use smart pointers to deal with dynamic memory. You can use just `std::vector<char>` as declaration type for your `buffer_`.

Reply:
ok